packages/expo-ui/android/src/main/java/expo/modules/ui/Utils.kt

```diff
@@ -59,11 +61,24 @@ val Color?.composeOrNull: androidx.compose.ui.graphics.Color?
 fun getImageVector(icon: String?): ImageVector? {
   if (icon.isNullOrEmpty()) return null
   return try {
-    val (theme, name) = icon.split(".")
-    val clazz = Class.forName("androidx.compose.material.icons.$theme.${name}Kt")
+    val clazz = Class.forName("androidx.compose.material.icons.${icon}Kt")
```

Comment:
I don't think this will work in production due to proguard stripping unused classes (also including all of them would lead to unnecessary bloat). See description here: https://github.com/expo/expo/pull/28845

A config plugin would be a solution to include only certain symbols in `keep.xml`, BUT that then makes the symbol pool static and doesn't allow you to add new ones in any way except rebuilding – I'd be OK with this limitation.

CC @alanjhughes

Reply:
@aleqsio, thanks for the heads-up here. Just to confirm, are you referring to the existing implementation or to my change?

If it's the latter, I'm happy to revert to that piece of code as it would still work as it was.